src/wms/src/test/java/org/geoserver/wms/featureinfo/RenderingBasedFeatureInfoTest.java

```diff
@@ -353,11 +353,30 @@ public void testMapWrapping() throws Exception {
             // with wrapping enabled we should get the giant polygon on the other side too
```

Comment:
I've noticed this file lacks a copyright header, could you add one while you're at it?